nexus/types/src/deployment.rs

```diff
@@ -1245,6 +1246,37 @@ pub enum PendingMgsUpdateDetails {
         /// expected contents of the inactive slot
         expected_inactive_version: ExpectedVersion,
     },
+    /// the RoT is being updated
+    Rot {
+        // implicit: component = ROT
+        // TODO-K: Is this accurate for the RoT as well?
+        // implicit: firmware slot id = 0 (always 0 for ROT)
```

Comment:
It's not, but I think you've already addressed this in this structure: `active_slot` tells you which one is currently active, so the other one is the inactive slot that needs to be updated.